src/openforms/scss/components/builder/_builder.scss

```diff
@@ -166,6 +166,10 @@ div.flatpickr-calendar.open {
     display: block;
     margin-left: 0;
     margin-right: 0;
+
+    ul.errorlist {
+      padding-left: 0px;
```

Comment:
Doesn't seem to break the error styling in the registration tab.

Reply:
does it break the styling of validation errors in the form slug field, for example?

Reply:
No, those ones have the class `module aligned` instead of `admin-fieldset`